docs/egyeb/uml-kod-kapcsolata/index_ger.md

```diff
@@ -1,25 +1,25 @@
 # Theorie der Beziehung zwischen dem UML-Klassendiagramm und dem Code
 
 Letztes Änderungsdatum: 2022.10.15  
```

Comment:
Ezt is inkább authors metaadattal jelezném és a módosítás dátumát sem kell így explicit kiírni, mert azt is figyeli az MKDocs